client/app/assets/less/main.less

```diff
@@ -11,7 +11,6 @@
 @import '~ui-select/dist/select.css';
 @import '~angular-toastr/src/toastr';
 @import '~angular-resizable/src/angular-resizable.css';
-@import '~pace-progress/themes/blue/pace-theme-minimal.css';
```

Comment:
Why?

Reply:
As far as I can tell, it's a dead link.

Reply:
It's not and it seems to be working for me with `MiniCssExtractPlugin`. I pushed a revert for this along with a few other fixes (see my commits in this PR). Once I see the build works -- will merge. 👍